src/App.js

```diff
@@ -1,17 +1,22 @@
-import React from "react";
+import React, { useState } from "react";
 import AddTodoForm from "./AddTodoForm";
 import TodoList from "./TodoList";
 
 const title = "Todo List";
+
 const App = () => {
-  const [newTodo, setNewTodo] = React.useState("");
+  const [todoList, setTodoList] = useState([""]);
```

Comment:
@wessstt here, you are initializing your toDoList to an array that has a string in it - Your initial toDolist should be an empty array [] so that you can push objects to it